jenkins/snapshot-elasticsearch.sh

```diff
@@ -47,29 +51,35 @@ function help () {
     )
     local ok=yes x
     for x in "${tools[@]}"; do which $x >/dev/null || ok=no; done
-    if test $ok = no
-    then
-        usage+=("${av0}: You need these tools on PATH: ${tools[*]}")
-    fi
+    test $ok = no && usage+=("${av0}: You need these tools on PATH: ${tools[*]}")
     ok=no
     for x in ${environments[*]}; do test $x = "$environment" && ok=yes; done
-    if test $ok = no
-    then
-        usage+=("${av0}: No environment: '$environment'")
-    fi
+    test $ok = no && usage+=("${av0}: No environment: '$environment'")
     ok=no
     for x in ${verbs[*]}; do test $x = "$verb" && ok=yes; done
-    if test $ok = no
-    then
-        usage+=("${av0}: No verb: '$verb'")
-    fi
+    test $ok = no && usage+=("${av0}: No verb: '$verb'")
     if test $ok = no
     then
         for line in "${usage[@]}"; do echo 1>&2 "$line"; done
         exit 1
     fi
 }
 
+# Echo ELASTICSEARCH_NODE_COUNT names of Elasticsearch nodes.
+#
+function elasticsearchNodes () {
+    local -i n last first=100
```

Comment:
the fact that the clusternodes are elasticsearch-<ENV>10X is sort of happenstance.  an artifact of puppet configuration we have in place.  IF I need to build a new node that is not exactly the same as current 10X nodes (like more mem, bigger disk, more CPU) - I must use a different number.  So it would a 20X cluster.  This was why I put in the elasticsearch1,2,3 DNS mappings.  So if today I built bigger servers - they will 201,202,203 and elasticsearch4,5,6.  so the assumption that first node is always 10X is not really valid.

Reply:
OK. I'll change first back to 1.